apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -304,6 +334,23 @@ const Ticket = new GQLListSchema('Ticket', {
                         }
                     }
                 }
+                const propertyChanged = resolvedData['property'] && resolvedData['property'] !== get(existingItem, 'property')
+                const propertyAddressChanged = resolvedData['propertyAddress'] && resolvedData['propertyAddress'] !== get(existingItem, 'propertyAddress')
+                const propertyAddressMetaChanged = resolvedData['propertyAddressMeta'] && resolvedData['propertyAddressMeta'] !== get(existingItem, 'propertyAddressMeta')
```

Comment:
the same for meta. we should update these fields automatically.

Reply:
Case when meta/address changes without property is when we change address of property itself...It cannot be detected here...That's why `afterChange` hook in property is used and that's why we're accepting 'controllable manual change'